ThingIFSDK/ThingIFSDK/Gateway/EndNode.swift

```diff
@@ -39,7 +39,7 @@ public struct EndNode: TargetThing, Equatable {
 extension EndNode: FromJsonObject {
 
     internal init(_ jsonObject: [String : Any]) throws {
-        guard let thingID = jsonObject["thingID"] as? String,
+        guard let thingID = jsonObject["endNodeThingID"] as? String,
```

Comment:
@naoki-akimoto @shinsuke-nara 
Sorry. That is my mistake. You are right. Then LGTM